molgenis-core-ui/src/main/javascript/modules/react-components/wrapper/Ace.js

```diff
@@ -34,7 +34,7 @@
 			};
 		},
 		getInitialState: function() {
-			return {value: this.props.value};
+			return {value: this.props.value || ''};
```

Comment:
Is this a bug fix? If yes, create a issue with seperate PR so that it will end up in the release notes and everyone is aware of this fix. If no, why this change?